docs/source/contrib/metrics.rst

```diff
@@ -34,35 +37,20 @@ Complete list of metrics:
 
 .. autosummary::
     :nosignatures:
-    :autolist:
-
-
-.. autoclass:: CanberraMetric
-
-.. autoclass:: FractionalAbsoluteError
-
-.. autoclass:: FractionalBias
-
-.. autoclass:: GeometricMeanAbsoluteError
-
-.. autoclass:: GeometricMeanRelativeAbsoluteError
-
-.. autoclass:: ManhattanDistance
-
-.. autoclass:: MaximumAbsoluteError
-
-.. autoclass:: MeanAbsoluteRelativeError
-
-.. autoclass:: MeanError
-
-.. autoclass:: MeanNormalizedBias
-
-.. autoclass:: MedianAbsoluteError
-
-.. autoclass:: MedianAbsolutePercentageError
-
-.. autoclass:: MedianRelativeAbsoluteError
-
-.. autoclass:: R2Score
-
-.. autoclass:: WaveHedgesDistance
```

Comment:
So, now, all these classes will have their one pages, and `docs/source/contrib/metrics.rst` will just hold a full list of them, right?

Reply:
Yes, they will be in generated/ dir when docs building

Reply:
Thanks!